config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -259,7 +258,6 @@ periodics:
     path_alias: sigs.k8s.io/cloud-provider-azure
     workdir: false
   spec:
-    serviceAccountName: prowjob-default-sa
```

Comment:
What was the reasoning behind dropping this? We dont need this anymore?

Reply:
I understood we only needed this to "simulate" the community infra service account config, and once we migrated this is the configuration we will expect by default.

Reply:
Its not needed in the community cluster, the default service account will work